packages/components/drawer/src/drawer-content.ts

```diff
@@ -0,0 +1,23 @@
+import { buildProps } from '@element-plus/utils'
+import { dialogContentProps } from '@element-plus/components/dialog'
+
+export const drawerContentProps = buildProps({
+  ...dialogContentProps,
+  direction: {
+    type: String,
+    default: 'rtl',
+    values: ['ltr', 'rtl', 'ttb', 'btt'],
+  },
+  withHeader: {
+    type: Boolean,
+    default: true,
+  },
+  size: {
+    type: [String, Number],
+    default: '30%',
+  },
+} as const)
+
+export const drawerContentEmits = {
+  close: () => true,
+}
```

Comment:
Please export type Props and Emits

Reply:
These props and emits is not available for our users, which I don't see the point exporting them.

Reply:
Many users may use Element Plus for secondary development, so they need to be provided with typings.

Reply:
This is not exposed globally how are they going to use this component anyways?